### DISCUSSION TITLE
Why do you use a wallet key for X-Api-Key when calling say "/usermanager/api/v1/users"?

### DISCUSSION THREAD

**Discussion by BenGWeeks** [lnbits/lnbits#2702] [General]
As per title, it's not always clear to me what we should be putting into X-Api-Key.

Calling `/usermanager/api/v1/users` with a `X-Api-Key` of the ID of a super user returns "Wallet not found" for example.

**Comment by motorina0** [comment]
It should be either `adminKey` (for write operations) or `inkey` (for read-only operations).
You can fetch these values from the wallet details.

![image](https://github.com/user-attachments/assets/961d652f-0abc-430e-97c0-1abfb92a6fdd)
